HopsanGUI/Dialogs/ComponentPropertiesDialog3.cpp

```diff
@@ -259,6 +260,14 @@ void ComponentPropertiesDialog3::applyPressed()
     setVariableValues();
 }
 
+void ComponentPropertiesDialog3::openSourceCode()
+{
+    auto& appearance = gpLibraryHandler->getModelObjectAppearancePtr(mpModelObject->getTypeName());
```

Comment:
This looks strange, the function returns a shared pointer. Here a reference to the shared pointer object is created. But then the temporary sharedpointer is destroyed (I assume). Then what does the reference refere to.
I think that the & should be removed.

Reply:
True. I will remove the &.